sssom/util.py

```diff
@@ -253,6 +263,11 @@ def filter_redundant_rows(
 
 
 def assign_default_confidence(df: pd.DataFrame) -> Tuple[pd.DataFrame, pd.DataFrame]:
+    """Assign numpy.NaN values to confidence that are blank.
```

Comment:
use intersphinx linking

```suggestion
Assign :data:`numpy.nan` to confidence that are blank.
```

Reply:
However, this string doesn't really make sense. The typing says this returns a tuple of two dataframes, and the docstring doesn't say at all what these are or how it's related to assigning default confidence